terraform/examples/gcp/nat_gateway/network.tf

```diff
@@ -0,0 +1,54 @@
+resource "google_compute_network" "firezone" {
```

Comment:
@jamilbk would it be better if we just accept a network ID and use `default` (that all projects have) if it's not set? I think if somebody deploys it to a VPC the network might already exist so creating a new one is not always desired.